packages/validator/src/cli-validator/utils/get-copyright-string.js

```diff
@@ -1,10 +1,10 @@
 /**
- * Copyright 2017 - 2023 IBM Corporation.
+ * Copyright 2017 - 2025 IBM Corporation.
  * SPDX-License-Identifier: Apache2.0
  */
 
 const getVersionString = require('./get-version-string');
 
 module.exports = function () {
-  return `IBM OpenAPI Validator (${getVersionString()}), @Copyright IBM Corporation 2017, 2024.`;
+  return `IBM OpenAPI Validator (${getVersionString()}), @Copyright IBM Corporation 2017, 2025.`;
```

Comment:
What do you think about using a calculated value for the year?

```suggestion
  return `IBM OpenAPI Validator (${getVersionString()}), @Copyright IBM Corporation 2017, ${new Date().getFullYear()}.`;
```

Reply:
I'm fine with that.

Reply:
I initially left it as a hard-coded year because I was thinking that we don't necessarily want the final copyright year to be the current year (i.e. we typically set the final year to the year when the most recent change was made).  But this applies more to copyrights in the code.  After thinking about it a little more, I agree that a command-line tool can always just display the current year as the final copyright year.